samples/Azure.ResourceManager.Sample/Generated/LogAnalyticsOperationResultContainer.cs

```diff
@@ -7,11 +7,11 @@
 
 namespace Azure.ResourceManager.Sample
 {
-    /// <summary> A class representing collection of LogAnalytics and their operations over a [ParentResource]. </summary>
-    public partial class LogAnalyticsContainer
+    /// <summary> A class representing collection of LogAnalyticsOperationResult and their operations over a [ParentResource]. </summary>
+    public partial class LogAnalyticsOperationResultContainer
```

Comment:
Why did this change from LogAnalyticsContainer to LogAnalyticsOperationResultContainer

Reply:
We mapped the resource name to "LogAnalyticsOperationResult" in readme.